slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc

```diff
@@ -0,0 +1,277 @@
+
+Before you try the examples, access Anypoint Studio (Studio) and verify that the *Mule Palette* view displays entries for Slack Connector. If not, follow the instructions in xref:slack-connector-studio.adoc#add-connector-to-project[Add the Connector to Your Mule Project].
+
+== Creating a Channel
+
+This example shows how to create a public Slack channel with a user-specified name.
+
+The following screenshot shows app flow for this example:
+
+.App flow for the Creating a Channel example
+image::slack-connector-create-channel-flow.png[This app flow shows the components used in the Creating a Channel example.]
+
+Implementing this example involves creating a new Mule project and configuring HTTP Listener, the Create Channel operation, two Transform Message components, and one Logger component.
+
+=== Prerequisites
+
+This example requires the `channels:write` scope in both the connection configuration and Slack application.
+
+=== Configure HTTP Listener
+
+Configure HTTP Listener to initiate a Mule flow when a call is made to the `/create-channel` path on localhost port `8081`:
+
+. Create a new Mule project in Studio.
+. In Studio, click *HTTP* and drag the *Listener* operation to the canvas.
+. On the Listener properties window, click *+* next to the *Connector configuration* field to add a global element.
+. Accept the defaults.
+. On the Listener properties window, set the *Path* field value to `/create-channel`.
+
+=== Add the First Transform Message Component
+
+This *Transform Message* component takes the channel name as input and then converts the name to JSON format:
+
+. In the *Mule Palette* view, select *Core* and drag the *Transform Message* component to the right of *Listener*.
+. Change the name of the *Transform Message* component to `Channel Details`.
+. Click the *Transform Message* component, add the channel name to the output, and set the output format to application/json:
+
+.First Transform Message component for the Creating a Channel example
+image::slack-connector-transform-channel-details.png[This Transform Message component specifies the channel name in JSON format.]
+
+=== Add the Create Channel Operation
+
+The *Create Channel* operation connects with Slack and posts a request to the `\https://slack.com/api/conversations.create` endpoint:
+
+. From the *Mule Palette* view, select *Slack* and drag the *Create Channel* operation to the right of *Listener*.
+. On the Search properties window, click *+* next to the *Connector configuration* field to add a global element.
+. Enter the following values:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Consumer Key | `${consumer.key}`
+|Consumer Secret| `${consumer.secret}`
+|Authorization url | `\https://slack.com/oauth/authorize`
+|Access token url | `\https://slack.com/api/oauth.access`
+|Scopes | `channels, enter write groups:write im:write mpim:write users:read.email`
```

Comment:
For Scopes, is the channels extraneous? Should it just be Enter the write groups: groups:write, im:write, mpim:write..." etc ?